src/Readers/AbstractReader.js

```diff
@@ -40,12 +40,13 @@ class AbstractReader {
    * @return {{__FILE__, __PATH__}}
    */
   parsePath(source) {
-
     const __FILE__ = path.basename(source);
     let __PATH__;
 
     let parsedURL = url.parse(source);
-    if (parsedURL.protocol) {
+
+    // url parse can find protocol in Windows-style path, so we check it
+    if (parsedURL.protocol && !parsedURL.protocol.match(/^(\w\:)?$/i)) {
```

Comment:
Independently of substitution ":" character said, that our filepath has protocol, and parsedURL.protocol becomes true. But maybe I didn't really understood what you meant.